### PR TITLE
Add broken tests for multiple appliers

### PR DESCRIPTION
**Pull request by jennybuckley**
only have one right now, this is to track what we need to fix before multiple appliers works

### CODE REVIEW

**Comment by jennybuckley** [comment]
@apelisse

File: merge/multiple_appliers_test.go [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+			},
+			Object: `
+				list:
+				- name: a
+				- name: c
+			`,
+			Managed: fieldpath.ManagedFields{
+				"apply-one": &fieldpath.VersionedSet{
+					Set: _NS(
+						_P("list", _KBF("name", _SV("a")), "name"),
```
Comment (apelisse) [inline merge/multiple_appliers_test.go]: And so we're considering also having `_P("list", _KBF("name", _SV("a")))` here, right?

Comment (jennybuckley) [reply merge/multiple_appliers_test.go]: Yes. When and if we make that change, we will have to change all of our tests.

**Comment by apelisse** [comment]
Cool, we probably need the same set of tests for "sets" rather than associative lists.

**Comment by apelisse** [comment]
and maps? I guess it also applies to maps.

**Pull request merged**